source/common/http/utility.h

```diff
@@ -131,9 +131,10 @@ Http1Settings parseHttp1Settings(const envoy::api::v2::core::Http1ProtocolOption
  * @param response_code supplies the HTTP response code.
  * @param body_text supplies the optional body text which is sent using the text/plain content
  *                  type.
+ * @param is_head_request tells if this is a HEAD request
```

Comment:
if this is a response to a HEAD request?

Reply:
Yes, I fix the syntax mistakes